src/shared/selinux-util.c

```diff
@@ -131,6 +225,10 @@ static int selinux_init(bool force) {
         if (!mac_selinux_use())
                 return 0;
 
+        r = dlopen_libselinux();
```

Comment:
This is redundant. when mac_selinux_use() is true, then the library is already loaded.